regexes/client/mobile_apps.yml

```diff
@@ -1908,7 +1909,7 @@
   version: '$1'
 
 # The free NRC app offers you the latest news and the digital version of the paper newspaper.
-- regex: '(?:NRC-Nieuws/|nl\.nrc\.nrcapp )([\d\.]+)'
+- regex: '(?:NRC-Nieuws/|nl.nrc.nrcapp |com.twipemobile.nrc )([\d\.]+)'
```

Comment:
regex: '(?:NRC-Nieuws/|nl\.nrc\.nrcapp |com\.twipemobile\.nrc )([\d\.]+)'

we just recently added escaping for any chars (I wouldn't want anyone to do double work again)

Reply:
Good to know!